.bazelrc

```diff
@@ -2,15 +2,18 @@
 # Import shared settings first so we can override below
 import %workspace%/.bazelrc.common
 
-## Disabled for now
 # Remote cache settings for local env
-# build --remote_cache=https://storage.googleapis.com/kibana-bazel-cache
-# build --incompatible_remote_results_ignore_disk=true
-# build --remote_accept_cached=true
-# build --remote_upload_local_results=false
+build --bes_results_url=https://app.buildbuddy.io/invocation/
+build --bes_backend=grpcs://cloud.buildbuddy.io
+build --remote_cache=grpcs://cloud.buildbuddy.io
+build --noremote_upload_local_results # Uploads logs & artifacts without writing to cache
```

Comment:
we can remove this line

Reply:
Pushed an update, but still getting the same thing: `Error loading timing profile. Make sure your cache is correctly configured.`

Reply:
Actually, that portion is required for read-only.